smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -223,27 +266,40 @@ contract DigitalaxAuction is Context, ReentrancyGuard {
         address winner = highestBid.bidder;
         uint256 winningBid = highestBid.bid;
 
-        // Ensure there is a winner
-        require(winner != address(0), "DigitalaxAuction.resultAuction: No one has bid");
+        // Ensure auction not already resulted
+        require(winningBid >= auction.reservePrice, "DigitalaxAuction.resultAuction: reserve not reached");
 
-        // FIXME check to ensure reserve is met
+        // Ensure there is a winner
+        require(winner != address(0), "DigitalaxAuction.resultAuction: no open bids");
 
-        // Clean up the highest winner
+        // Clean up the highest winner (N.b. dont clean up auction mapping as this NFT cannot only be auctioned once)
         delete highestBids[_garmentTokenId];
 
-        // FIXME delete auction too (maybe)?
-
-        // TODO check this feature against the spec? Does this need to be on-chain?
         // Record the primary sale price for the garment
         garmentNft.setPrimarySalePrice(_garmentTokenId, winningBid);
 
-        // Send the designer their earnings
-        (bool designerTransferSuccess,) = garmentNft.garmentDesigners(_garmentTokenId).call{value : winningBid}("");
-        require(designerTransferSuccess, "DigitalaxAuction.resultAuction: Failed to send the designer their royalties");
+        if (winningBid > auction.reservePrice) {
```

Comment:
we have a require above which covers this `winningBid >= auction.reservePrice`. We are also doing a different equality check